### ISSUE TITLE
Submission of xxx failed, already exists!

### ISSUE THREAD

**Issue by blueoinkhead** [bloocoin/bloocoin-client#2]
I thought each miner was assigned their own "coin" to mine on. But i've mined 2 coins and get this error when i update from the server.

Although im using BLC Client (Latest version). I think this is a huge fail, and you could probaly just guess names. And send coins to yourself..

I can see the coins in my wallet, but they might be duplicated.. If someone mined the same ones..
![blc client](https://f.cloud.github.com/assets/4381238/480325/6d2e5218-b82b-11e2-8650-4445daaddf1d.png)

**Comment by f-prime** [comment]
No, miners look for solutions without really interesting with the server. This is common especially now that we are nearing the stage when a difficulty increase will be happening soon (in a bout 100,000 coins)

**Comment by ajvpot** [comment]
Is anyone going to close this?
